shared/components/_button.scss

```diff
@@ -1,8 +1,33 @@
 // Base
 @mixin nio-button--colorize($background, $foreground) {
+  $button-animation: unique-id();
+  $ghost: transparentize($background, 0.5);
```

Comment:
i think it may be better to promote this to a mixin param.

```scss
@mixin nio-button--colorize($background, $foreground, $ghost: transparentize($background, 0.5)) { }
```